fenix/app/src/main/java/org/mozilla/fenix/settings/SecretSettingsFragment.kt

```diff
@@ -67,6 +67,12 @@ class SecretSettingsFragment : PreferenceFragmentCompat() {
         requirePreference<Preference>(R.string.pref_key_custom_sponsored_stories_parameters).apply {
             isVisible = Config.channel.isNightlyOrDebug
         }
+
+        requirePreference<SwitchPreference>(R.string.pref_key_enable_tabs_tray_to_compose).apply {
+            isVisible = Config.channel.isNightlyOrDebug
```

Comment:
Just double-checking that we want this secret setting available in nightly as well? Seems fine to me since it's hidden

Reply:
Yeah, this was intentional. That way UX et all can access the changes more easily.